index.html

```diff
@@ -61,6 +61,324 @@ <h1 class="display-5 fw-bold primary-text">I’m Yannick<br> Glad to see you!</h
       </div>
   </section>
 
+  <section class="work-section section-pd-50-50 section-mg-50-0">
+
+     <div class="container">
+
+        <div class="flex-container">
+          <div class="flex-item-left">
+              
+              <img src="image/portfolio1.png" class="img-responsive" alt="image-work1"></div>
+          
```

Comment:
- [ ] Images should be background images not `img` tag. Kindly make this and other images background images.